### ISSUE TITLE
508/a11y: Sitewide - links to PDFs should have indication to users

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#343]
This is a sitewide issue, as there are many links to PDF documents throughout the site with no indication that the link goes to a PDF document.

Note: This is a well-known usability issue closely related to accessibility. While it is not a strict failure of any WCAG/508 success criteria, people with disabilities are disproportionately affected by this problem.

Here is some text from [WebAIM about this problem](https://webaim.org/techniques/hypertext/hypertext_links#non_html):

> Users should generally be alerted to links that lead to non-HTML resources, such as PDF files, Word files, PowerPoint files, and so on. However, there is some debate as to whether the content author or the browser should be the one to alert the user. The trouble is that none of the browsers or screen readers currently alert the user at all, so the debate is more theoretical than practical.

In the WAVE testing tool from WebAIM, PDF links have the following information highlighted for attention of content authors:

> ### Alerts
> Link to PDF document
> 
> ### What It Means
> A link to a PDF document is present.
> 
> ### Why It Matters
> Unless authored with accessibility in mind, PDF documents often have accessibility issues. Additionally, PDF documents are typically viewed using a separate application or plug-in, and can thus cause confusion and navigation difficulties.
> 
> ### How to Fix It
> Ensure the PDF document is natively accessible. Additionally, inform the user that the link will open a PDF document. Because PDF documents may have limitations in accessibility (particularly for complex content) and require a separate program, HTML content should often be used in place of or in addition to the PDF document.

## Recommendation:

From WebAIM:
When identifying the link file type, this additional content should be presented inside the link, rather than just after it, so that the information is presented with the link if a screen reader users navigates by links or reads a link list.

Use
`<a href="1040.pdf">Tax form 1041 (PDF)</a>`
rather than
`<a href="1040.pdf">Tax form 1041</a> (PDF)`

**Comment by alyssagalik** [comment]
Updated on these pages if you have any feedback on the format. 

- https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/methods/
- https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/toolkits/
- https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/projects/wildfire-risk-assessments/

**Comment by iamjolly** [comment]
@alyssagalik - those look great!
